tests/document_ops.rs

```diff
@@ -508,6 +513,43 @@ async fn doc_create_without_self_grant() -> Result<(), IronOxideErr> {
     Ok(())
 }
 
+#[tokio::test]
+async fn doc_create_shared_user_can_revoke() -> Result<(), IronOxideErr> {
+    let (user1, user1_sdk) = init_sdk_get_user().await;
+    let (user2, user2_sdk) = init_sdk_get_user().await;
+
+    let doc = [0u8; 64];
+
+    let doc_result = user1_sdk
+        .document_encrypt(
+            doc.into(),
+            &DocumentEncryptOpts::with_explicit_grants(
+                Some(create_id_all_classes("").try_into()?),
+                Some("first name".try_into()?),
+                true,
+                vec![UserOrGroup::User { id: user2.clone() }],
+            ),
+        )
+        .await?;
+
+    // should be a user with access, but not the currently initd user
+    assert_eq!(doc_result.grants().len(), 2);
```

Comment:
What does this comment mean?

Reply:
It's left over from the base test. Removed.